### PR TITLE
OOO Users Enhancement

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32075

## Description
Added a new argument to the GetUsersOnCall command that allows excluding users that are Out Of Office.

### CODE REVIEW

File: Packs/ShiftManagement/Scripts/GetUsersOnCall/GetUsersOnCall.py [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
+def filter_OOO_users(get_users_response):
+    """
+    Given the response with all OnCall users, remove the users that are Out Of Office, using the list `OOO List`.
+    """
+    all_users = get_users_response.get('Contents')
+    if not all_users:
+        return 'No data returned'
+
+    OOO_users_list = demisto.executeCommand('getList', {'listName': 'OOO List'})
+    if is_error(OOO_users_list):
```
Comment (yuvalbenshalom) [inline Packs/ShiftManagement/Scripts/GetUsersOnCall/GetUsersOnCall.py:18]: are we getting an error if the list dose not exist?
if so, I like to split it.
if list dose not exist, log debug that it dose not exist.
for all others, log error.

**Review by yuvalbenshalom — COMMENTED** [review]
looks good. see my comment

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/ShiftManagement/ReleaseNotes/1_1_4.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### GetUsersOnCall
+- Added a new argument *include_OOO_users* which allows the user to choose if to include in the output also users
+ that are Out Of Office. By default, Out Of Office users are not included in the output.
+- Updated the docker image from `3.8.5.10845` to `3.9.1.14969`.
```
Comment (amshamah419) [inline Packs/ShiftManagement/ReleaseNotes/1_1_4.md:6]: The convention should be `Upgraded the Docker image to demisto/pyjwt3:1.0.0.8871.`

File: Packs/ShiftManagement/ReleaseNotes/1_1_4.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### GetUsersOnCall
+- Added a new argument *include_OOO_users* which allows the user to choose if to include in the output also users
+ that are Out Of Office. By default, Out Of Office users are not included in the output.
```
Comment (amshamah419) [inline Packs/ShiftManagement/ReleaseNotes/1_1_4.md:5]:
```suggestion
- Added the new argument *include_OOO_users* which allows users to choose whether to include in the output users that are Out Of Office. By default, Out Of Office users are not included in the output.
```

**Pull request merged**